src/complexity_measures/statistical_complexity.jl

```diff
@@ -7,7 +7,7 @@ export StatisticalComplexity, entropy_complexity, entropy_complexity_curves
     StatisticalComplexity <: ComplexityEstimator
     StatisticalComplexity([x]; kwargs...)
 
-An estimator for the statistical complexity and entropy according to Rosso et al. (2007)[^Rosso2007](@ref),
+An estimator for the statistical complexity and entropy according to Rosso et al. (2007)[^Rosso2007],
 used with [`complexity`](@ref).
 
 ## Keyword arguments
```

Comment:
The docstring doesn't say anywhere that different entropies are possible or how they should be given as input.

Reply:
> The docstring doesn't say anywhere that different entropies are possible or how they should be given as input.

Nice catch. Yes, there should be another line in the list of keyword arguments, which also specifies what the default is (I guess `Shannon(base = 2)`?)